clang/utils/perf-training/bolt.lit.cfg

```diff
@@ -20,9 +21,13 @@ elif clang_bolt_mode == "perf":
 else:
     assert 0, "Unsupported CLANG_BOLT_MODE variable"
 
-config.clang = perf_wrapper + os.path.realpath(
+clang_nowrapper = os.path.realpath(
     lit.util.which(clang_binary, config.clang_tools_dir)
 ).replace("\\", "/")
+config.clang = perf_wrapper + clang_nowrapper
```

Comment:
Just for my understanding: we're keeping perf_wrapper set if CLANG_BOLT=perf. However we pass perf_wrapper in front of cmake command in `clang/utils/perf-training/llvm-support/build.test` – do we end up with nested perf invocations, or do we strip perf_wrapper from clang in this case?